app/models/learning.rb

```diff
@@ -1,12 +1,20 @@
 # frozen_string_literal: true
 
 class Learning < ActiveRecord::Base
-  enum status: { started: 0, complete: 1, not_complete: 2 }
+  enum status: { not_complete: 0, started: 1, submitted: 2, complete: 3 }
   belongs_to :user, touch: true
   belongs_to :practice
 
   validates :user_id, presence: true
   validates :practice_id,
     presence: true,
     uniqueness: { scope: :user_id }
+  validate :startable_practice
+
+  private
+    def startable_practice
+      if Learning.started.where(user_id: user_id).where.not(id: id).exists? && started?
```

Comment:
```suggestion
      if started? && Learning.started.where(user_id: user_id).where.not(id: id).exists?
```

の方が効率が良いです。

`started?`がfalseの時にSQLを実行する必要はありませんので

Reply:
重ね重ねありがとうございます!!

修正いたしました。